<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View_Inicial</class>
 <widget class="QMainWindow" name="View_Inicial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(80, 80, 80)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Titulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>801</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Informacion del Hacker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Img_Hacker">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;app\views\UI\resources\IMG_7510.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Nombre">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:  Es7eB4N_Murcia_Saav3dra.exe</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Edad">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edad:  0x1A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Nacionalidad">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nacionalidad:  Colombia</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Recompensa">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RECOMPENSA: ₿ 5.2 BTC por información que lleve a su captura.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Motivo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Motivo: El sujeto conocido como 3$73B4N |/|URC14  está desarrollando un sistema de encriptación</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Informacion">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre: Esteban Murcia Saavedra&lt;/p&gt;&lt;p&gt;Materia: Aplicaciones I&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Encriptar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
     </font>
    </property>
    <property name="text">
     <string>Encriptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Decepcriptar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
     </font>
    </property>
    <property name="text">
     <string>Decepcriptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Inicio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
     </font>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
